src/signals/my-incidents/components/EmailInput.tsx

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { useRef, useState } from 'react'
+
+import { useHistory } from 'react-router-dom'
+import * as yup from 'yup'
+
+import Button from 'components/Button'
+import Label from 'components/Label'
+
+import { useMyIncidentContext } from '../context'
+import { routes } from '../definitions'
+import { usePostEmail } from '../hooks'
+import { ErrorWrapper, StyledErrorMessage, StyledInput } from './styled'
+import type { Error } from './types'
+
+const schema = yup.string().email().required().max(254)
+
+const validateInput = async (
+  inputRef: React.RefObject<HTMLInputElement>,
+  setValidation: (error: Error) => void
```

Comment:
this is actually setting the error derived from the validation.
consider changing the name setError

Reply:
Removed the whole component since it is not used.